core/blockchain.go

```diff
@@ -322,8 +322,8 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, genesis *Genesis
 		// for it to be able to recover if interrupted during the transition
 		// but that's left out to a later PR since there's not really a need
 		// right now.
-		bc.stateCache.InitTransitionStatus(true, true)
-		bc.stateCache.EndVerkleTransition()
+		bc.stateCache.InitTransitionStatus(false, false)
+		// bc.stateCache.EndVerkleTransition()
```

Comment:
This is a required change to avoid an error when processing the second block.
Before merging we have to figure out a coherent way to do this for both replay and non-replay setups to work correctly.

Reply:
indeed. but if it's the only change that we have to keep in a separate branch, that's ok too.